storybookjs/storybook
# Addon-viewports: Fix initial load state

Issue: https://github.com/storybookjs/storybook/issues/11548

## What I did
- CHANGE code to respect the existing state in useAddonState

## how to test
- load official example
- go to `/?path=/story/addons-viewport-custom-default-kindle-fire-2--overridden-via-with-viewport-parameterized-decorator`

expected: the story loads in the 'ipad' viewport immediately
